README.md

```diff
@@ -1,3 +1,31 @@
+* *Status:* Production
```

Comment:
It would be nice to get confirmation from Lisa. All of the templates I created didn't put the stuff right at the top either. Also, her example doesn't put it straight at the top, but it's possible that it because it is enveloped in the rest of the document. So it is unclear exactly how they are matching on these tags.